gmplot/utility.py

```diff
@@ -367,7 +367,7 @@ def _pretty_format_markdown(directory):
         # For each parameter line...
         _PARAMETER_REGEX = '(%s)(%s)(%s)' % (
             ' *.*? ', # Matches whatever comes before the type, like: '''  * **origin** '''
-            '\(.*\)', # Matches whatever makes up the type, like:     '''(*(**float**, **float**)*)'''
+            r'\(.*\)', # Matches whatever makes up the type, like:     '''(*(**float**, **float**)*)'''
             ' – .*'   # Matches whatever comes after the type, like:  ''' – Origin, in latitude/longitude.'''
```

Comment:
Thanks for the fix! Just a tiny nit, could you line up the '_# Matches whatever..._' comments across the three lines?

Reply:
Fixed with 4504e4685bc924d91cf8e79b1cfcda58d375f137